spec/plugin.spec.js

```diff
@@ -221,4 +221,48 @@ describe('ManifestPlugin', function() {
       });
     });
   });
+
+  describe('assets added by third parties to "compilation.assets"', function() {
+    it('are included also when enabled.', function(done) {
+
+      var options = webpackConfig({
```

Comment:
Can you not use `webpackConfig`?
Follow the same pattern as `with ExtractTextPlugin`